extensions/shared/components/block-styles-selector/index.js

```diff
@@ -0,0 +1,96 @@
+/**
+ * External Dependencies
+ */
+import classnames from 'classnames';
+import { isEqual } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+import { memo } from '@wordpress/element';
+import { getBlockType, getBlockFromExample, createBlock } from '@wordpress/blocks';
```

Comment:
`getBlockFromExample` isn't supported on WordPress 5.2, so that won't work:

<img width="307" alt="screenshot 2020-01-29 at 16 38 22" src="https://user-images.githubusercontent.com/426388/73371413-f418c800-42b5-11ea-956b-412b19ad84fb.png">
<img width="323" alt="screenshot 2020-01-29 at 16 38 07" src="https://user-images.githubusercontent.com/426388/73371414-f4b15e80-42b5-11ea-96a2-46d2a2b609b4.png">

Related: #14469

Reply:
@jeherve this is the intended behaviour. For users who don't have these more recent additions to core, we just show a button, as in your screenshot.